src/Service/UpdaterWorker.php

```diff
@@ -72,7 +72,7 @@ public function __construct(
     }
 
     /**
-     * @return array{status: string, message?: string, after?: \DateTime, vendor?: string, details?: string, e?: \Exception|\Throwable}
+     * @return array{status: string, message?: string, after?: \DateTimeInterface, vendor?: string, details?: string, e?: \Exception|\Throwable}
```

Comment:
You missed that I dropped `Exception` here, it's AFAIK included in Throwable so redundant.